maintenance/ebpf/enabling-bpf.md

```diff
@@ -49,33 +55,34 @@ eBPF (or "extended Berkeley Packet Filter"), is a technology that allows safe mi
 eBPF mode has the following pre-requisites:
 
 - A supported Linux distribution:
-  
-  - Ubuntu 20.04.
-  - Red Hat v8.2 with Linux kernel v4.18.0-193 or above (Red Hat have backported the required features to that build).
-  - Another [supported distribution]({{site.baseurl}}/getting-started/kubernetes/requirements) with Linux kernel v5.3 or above.
-    
+
+    - Ubuntu 20.04.
```

Comment:
FYI, https://wiki.ubuntu.com/BionicBeaver/ReleaseNotes#Kernel

> Ubuntu 18.04.4 ships with a v5.3 based Linux kernel updated from the v5.0 based kernel in 18.04.3.

Dunno if you want to update the 20.04 bullet with that?